modin/engines/base/io/column_stores/parquet_reader.py

```diff
@@ -12,6 +12,7 @@
 # governing permissions and limitations under the License.
 
 import os
+import s3fs
```

Comment:
@prutskov @Garra1980 doesn't this effectively make `s3fs` an always-required dependency?

Reply:
@vnlitvinov this already fixed in #2326